Readme.md

```diff
@@ -95,6 +95,32 @@ Optionally, the following environment variables can be set:
 
 Once configured, pipelines can be migrated to GitHub Actions by opening an issue with the relevant issue template and following the instructions.
 
+### Custom transformers
+
+See [./transformers/README.md](./transformers/README.md) for details on using custom transformers for `dry-run` and `migrate` commands.
```

Comment:
Looks like this PR doesn't have a `./transformers/README.md` file -- did you mean to include that as well?

Reply:
Good catch, this line should be removed altogether